cmd/neofs-node/object.go

```diff
@@ -739,3 +738,52 @@ func (c *cfg) IsLocalNodePublicKey(b []byte) bool { return c.IsLocalKey(b) }
 func (c *cfg) GetNodesForObject(addr oid.Address) ([][]netmapsdk.NodeInfo, []uint, error) {
 	return c.cfgObject.containerNodes.getNodesForObject(addr)
 }
+
+// GetContainerNodes reads storage policy of the referenced container from the
+// underlying container storage, reads current network map from the underlying
+// storage, applies the storage policy to it, gathers storage nodes matching the
+// policy and returns sort interface.
+//
+// GetContainerNodes implements [putsvc.NeoFSNetwork].
+func (c *cfg) GetContainerNodes(cnrID cid.ID) (putsvc.ContainerNodes, error) {
```

Comment:
why isn't it a part of `containerNodes`? our `cfg` becomes too complex. this method only uses `cfg`'s `c.cfgObject.containerNodes`